### ISSUE TITLE
lack of epoch_fit()

### ISSUE THREAD

**Issue by liugd18** [gasking/ASDA#1]
![image](https://github.com/user-attachments/assets/789bae04-5288-488d-aa8a-43ca8cdc25f4)
as shown above, there is no content in utils_fit.py, which brought me great trouble
is the project incomplete?

**Comment by promisedong** [comment]
Yes, I've been refactoring the code recently so this part is not open sourced.
